extensions/amp-viewer-integration/0.1/amp-viewer-integration.js

```diff
@@ -90,7 +90,7 @@ export class AmpViewerIntegration {
       return Promise.resolve();
     }
 
-    if (MOVE_FIXED_LAYER) {
+    if (MOVE_FIXED_LAYER || IS_SXG) {
```

Comment:
You don't need to do this for `IS_SXG` (as this code is part of an extension injected only when the document is served in a viewer from the AMP cache)

Reply:
I deleted this change.